heapster-saw/examples/Makefile

```diff
@@ -34,7 +34,7 @@ rust_lifetimes.bc: rust_lifetimes.rs
 	rustc --crate-type=lib --emit=llvm-bc rust_lifetimes.rs
 
 # Lists all the Mr Solver tests, without their ".saw" suffix
-MR_SOLVER_TESTS = arrays_mr_solver linked_list_mr_solver sha512_mr_solver
+MR_SOLVER_TESTS = # arrays_mr_solver linked_list_mr_solver sha512_mr_solver
```

Comment:
Worth filing an issue about this as a reminder.

Reply:
Done in #1782